tfhe/src/core_crypto/commons/traits/create_from.rs

```diff
@@ -18,7 +18,7 @@ impl<'data, T> CreateFrom<&'data [T]> for DummyCreateFrom {
     type Metadata = ();
 
     #[inline]
-    fn create_from(_: &'data [T], _: Self::Metadata) -> Self {
+    fn create_from(_: &'data [T], (): Self::Metadata) -> Self {
```

Comment:
What was the name of the lint for this ? I want to know the rational behind this, `_` is the prefix used for intentionnally unused variables

Reply:
I have to agree this looks weird, maybe use `_` and `__` to avoid arguments seemingly having the same name

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/ignored_unit_patterns

Reply:
As discussed at the office, I don't think we want this lint, plus the rationale to keep it is "it will allow to detect type changes" does not bring much value as we are throwing the data away anyways, we are not at a point where changing APIs/ABIs is really a concern

Reply:
Unfixed